widget_pages/patient_form.py

```diff
@@ -67,57 +65,186 @@ class PatientFormWindow(QWidget):
+        self.patientFirstNameLineEdit.setStyleSheet(
+            "background-color: #f5f5f5; height: 40px; border-radius: 10px; padding: 0px 10px"
         )
+        self.patientNameLayout.addWidget(self.patientFirstNameLineEdit)
+
+        self.patientLastNameLineEdit = QLineEdit()
+        self.patientLastNameLineEdit.setContentsMargins(10, 0, 30, 0)
+        self.patientLastNameLineEdit.setPlaceholderText("Last Name")
+        self.patientLastNameLineEdit.setFont(QFont("Avenir", 18))
+        self.patientLastNameLineEdit.setStyleSheet(
+            "background-color: #f5f5f5; height: 40px; border-radius: 10px; padding: 0px 10px"
+        )
+        self.patientNameLayout.addWidget(self.patientLastNameLineEdit)
+        self.patientFormLayout.addLayout(self.patientNameLayout)
+        
+        self.genderLabel = Label("Gender/Sex")
+        self.genderLabel.setContentsMargins(30, 10, 0, 0)
+        self.patientFormLayout.addWidget(self.genderLabel, alignment=Qt.AlignmentFlag.AlignBottom)
+
+        self.genderLayout = QHBoxLayout()
+        self.genderLayout.setContentsMargins(30, 0, 0, 0)
+
+        self.radioButton = QRadioButton("Male")
+        self.radioButton.setContentsMargins(0, 0, 0, 0)
+        self.radioButton.setChecked(True)
+        self.radioButton.gender = "Male"
+        self.radioButton.setFont(QFont("Avenir", 18))
+        self.radioButton.toggled.connect(self.selectedGenderType)
+        self.genderLayout.addWidget(self.radioButton)
+
+        self.radioButton = QRadioButton("Female")
+        self.radioButton.setContentsMargins(10, 0, 0, 0)
+        self.radioButton.setFont(QFont("Avenir", 18))
+        self.radioButton.gender = "Female"
+        self.radioButton.toggled.connect(self.selectedGenderType)
+        self.genderLayout.addWidget(self.radioButton)
+
+        self.patientFormLayout.addLayout(self.genderLayout)
+
+        self.birthdayLayout = QVBoxLayout()
+        self.birthdayLayout.setContentsMargins(30, 0, 30, 0)
+        
+        self.birthdayLabel = Label("Brithday")
+        self.birthdayLabel.setContentsMargins(0, 10, 0, 0)
+        self.birthdayLayout.addWidget(self.birthdayLabel, alignment=Qt.AlignmentFlag.AlignBottom)
 
-        self.weightLabel = Label("Weight (kg)")
-        self.weightEdit = LineEdit("kg")
-        self.patientFormLayout.addWidget(FormRow(self.weightLabel, self.weightEdit))
-
-        self.heightLabel = Label("Height (cm)")
-        self.heightEdit = LineEdit("cm")
-        self.patientFormLayout.addWidget(FormRow(self.heightLabel, self.heightEdit))
+        self.birthdayEdit = QDateEdit()
+        self.birthdayEdit.setContentsMargins(0, 0, 0, 0)
+        self.birthdayEdit.setFont(QFont("Avenir", 15))
+        self.birthdayLayout.addWidget(self.birthdayEdit)
+        self.patientFormLayout.addLayout(self.birthdayLayout)
+
+        self.bodyLayout = QGridLayout()
+        self.bodyLayout.setContentsMargins(0, 0, 0, 0)
+
+        self.weightLabel = Label("Weight(kg)")
```

Comment:
Can we have a space in between weight and (kg) please, same as all the other labels that have this issue.

Reply:
done